src/ui/OverlayController.tsx

```diff
@@ -119,10 +137,13 @@ export class OverlayController extends React.Component<IControllerProps, void> {
           />
         </div>
         <div className="mdl-card__menu">
-          {remote.isEnabled ? <a onClick={toggleShareMenu}>SHARED</a> : null}
+          {remote.isEnabled ?
+            <a onClick={() => this.toggleShareMenu()}>SHARED</a>
```

Comment:
It's a bit weird that you have the ? at the end of one line and the : at the beginning of the other.  I'd pick one style and stay consistent.